packages/compass-aggregations/src/components/focus-mode/focus-mode-modal-header.spec.tsx

```diff
@@ -57,15 +57,17 @@ describe('FocusModeModalHeader', function () {
     const context = renderFocusModeModalHeader();
 
     expect(
-      await context.findByLabelText('Edit previous stage')
-    ).to.have.attribute('disabled');
+      (await context.findByLabelText('Edit previous stage')).getAttribute(
+        'aria-disabled'
+      )
+    ).to.equal('true');
   });
 
   it('disables "next stage" button when showing last stage', async function () {
     const context = renderFocusModeModalHeader({ stageIndex: 2 });
 
     expect(await context.findByLabelText('Edit next stage')).to.have.attribute(
-      'disabled'
+      'aria-disabled'
```

Comment:
You might want to start checking for attribute value here because compared to `disabled`, this attribute is always present.